### PR TITLE
created html file and updates ppt file

### PR DESCRIPTION
**Pull request by abdulhaseebshaik**
created html file and updates ppt file

### CODE REVIEW

**Comment by IshaaqhShaik** [comment]
why is .dsstore added here ?

File: Module-2/Day-5/.DS_Store
Comment (IshaaqhShaik) [inline Module-2/Day-5/.DS_Store:1]: why is this file added ?

**Review by IshaaqhShaik — CHANGES_REQUESTED** [review]

File: Module-2/Day-5/.DS_Store
Comment (abdulhaseebshaik) [reply Module-2/Day-5/.DS_Store:1]: Ds_store file deleted

**Review by IshaaqhShaik — APPROVED** [review]
